counterpartylib/lib/config.py

```diff
@@ -7,7 +7,7 @@
 # Versions
 VERSION_MAJOR = 9
 VERSION_MINOR = 55
-VERSION_REVISION = 1
+VERSION_REVISION = 2
```

Comment:
A reparse is a significant interruption for those of us running counterparty in production.  It brings down a production counterparty server for hours.